src/Model/Survey/Survey.cs

```diff
@@ -1,28 +1,154 @@
-namespace Survey;
+using System;
+namespace Model.Survey;
 
-using Question;
-using Answer;
+using Model.Question;
+using Model.Answer;
+using System.Collections.Generic;
 
-public class Survey  {
-    // private int surveyID;
-    // private Question[][] surveyQuestions;
-    // public IGetQuestion GetNextQuestion {get;}
+public class Survey : IReadOnlySurvey, IModifySurvey {
 
-    // public IGetQuestion GetQuestionA {get;}
+    public IReadOnlyAnswer ReadOnlyAnswer {get;}
     
-    // public IGetQuestion GetQuestionB {get;}
+    public IModifyAnswer ModifyAnswer {get; set;}   
+    private int current;    
+    public int surveyId {get;}
```

Comment:
Public attribute should be pascal case
```suggestion
    public int SurveyId {get;}
```